graphite/graphite.go

```diff
@@ -94,7 +94,7 @@ func (p *Parser) Parse(line string) (influxdb.Point, error) {
 		return influxdb.Point{}, err
 	}
 
-	timestamp := time.Unix(0, unixTime*int64(time.Millisecond))
+	timestamp := time.Unix(unixTime, 0)
```

Comment:
Just tested sending fractional seconds to a real graphite instance and it does accept them.  Don't think this is fully correct as is.  Probably need to use `strconv.ParseFloat` instead.